### PR TITLE
latest windows makefile and requirements

### PR DESCRIPTION
**Pull request by voisardf**
latest config on windows Neuchâtel

### CODE REVIEW

**Comment by voisardf** [comment]
This branch is manly designed to resolve issues for other Windows using cantons. Do not merge for now

**Comment by jwkaltz** [comment]
@voisardf Travis tests are OK now (after a restart), feel free to merge

File: requirements-windows.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-wheels/Shapely-1.6.4.post1-cp27-cp27m-win32.whl
+wheels/Shapely-1.6.4.post1-cp36-cp36m-win_amd64.whl
+numpy==1.16.3  # rq.filter: <1.17.0
```
Comment (voisardf) [inline requirements-windows.txt]: I think, that this version will get uninstalled and overwritten later on, as it's also defined in the requirements,txt. So it has to be reinstalled manually afterwards or adapted in the requirements.txt before building the application.

**Comment by voisardf** [comment]
Merging as discussed with @jwkaltz

**Pull request merged**